.github/ISSUE_TEMPLATE/bug_report_port.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Bug report (hardware / driver / port)
+about: Report an issue in a port or driver
+title: ''
+labels: ports
+assignees: ''
+
+---
+
+**For questions about MicroPython or for help using MicroPython, please see the MicroPython Forum -- https://forum.micropython.org/**
+
+**Issue description**
+A clear and concise description of what the bug is.
+
+If this issue involves external hardware, please include links to relevant datasheets and schematics.
+
+**MicroPython port & version**
+Please provide at least one of:
+ - Git commit hash and port/board
+ - Pre-built firmware filename
+ - Version information shown in the REPL (hit Ctrl-B)
+
+**Additional context**
+Add any other context about the problem here.
```

Comment:
What is still missing is asking for build platform and environment. We need a way to counter "MicroPython doesn't build" or "Custom MicroPython build doesn't do XXX" reports which don't mention which port is being built, for which target platform, the compiler used, the build command line, the relevant build output etc. Or perhaps that needs a separate template?